test/fast/Unit tests/nvm_curl_use_compression

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+
+cleanup () {
+  unset -f die
+}
+
+die () { echo -e "$@" ; cleanup ;  exit 1; }
+
+if type "curl" > /dev/null 2>&1 ; then
+  curl_exec="$(type "curl")"
+  sudo rm -rf "${curl_exec}"
```

Comment:
i don't think this is necessary (or safe, since this runs on developer's systems); the function below should mask it sufficiently.